components/atom/spinner/src/index.js

```diff
@@ -11,6 +11,8 @@ const TYPES = {
 
 const DELAY = 500 // ms
 const BASE_CLASS = 'sui-AtomSpinner'
+const CLASS_FULL = `${BASE_CLASS}--fullPage`
+const CLASS_NO_BACKGROUND = `${BASE_CLASS}-noBackground`
```

Comment:
I guess this should be a modifier with two `--`.

Reply:
Fixed!